### PR TITLE
Added Infinite scroll on Explore Page

### LINKED ISSUE #29: Infinite Scrolling

### PR DESCRIPTION
**Pull request by romabulani**

### CODE REVIEW

File: src/backend/controllers/PostController.js [hunk 1]
```diff
@@ -292,3 +292,21 @@ export const deletePostHandler = function (schema, request) {
...
   }
 };
+
+/**
+ * This handler handles gets all posts in the db.
+ * send GET Request at /api/posts/page/pageNum
+ * */
+//=>0,1,2,3
+//1=>0,1,2,3,4,5,6,7
+//2=>0, 1,2,3,4,5,6,7,8,9,10,11
```
Comment (Rohanmond) [inline src/backend/controllers/PostController.js:302]: remove these comments

**Review by Rohanmond — APPROVED** [review]
code looks good

File: src/backend/controllers/PostController.js (hunk 1 above)
Comment (romabulani) [reply src/backend/controllers/PostController.js:302]: I have Kept this for future reference/explanation of pageNum

**Review by shrey27 — APPROVED** [review]
Code looks good
UI is working expected
PR is ready to merge

**Pull request merged**